components/footer.js

```diff
@@ -179,7 +179,7 @@ const Footer = ({
             <Service href={`mailto:${email}`} icon="email-fill" />
           </Grid>
           <Text my={2}>
-            <Link href="tel:1-855-625-HACK">1-855-625-HACK</Link>
+            <Link href="tel:+1 (844) 237 2290">+1 (844) 237 2290</Link>
```

Comment:
Great! In addition to @sampoder's feedback, can we also change the phone number in the `href` to use hyphens as the separator?

This is because the [spec](https://datatracker.ietf.org/doc/html/rfc3966#section-5.1.1) says `tel` URIs should not contain spaces.

The inner content of the link can stay the same!


```suggestion
            <Link href="tel:+1-844-237-2290">+1 (844) 237 2290</Link>
```

Reply:
Sure @garyhtou